src/pages/learn.js

```diff
@@ -211,14 +240,37 @@ export default ({
                     <FeaturedArticles articles={featuredArticles} />
                 </HeaderContent>
             </Header>
-            <Article>
+
+            <TabBar
+                handleClick={setActiveItem}
+                leftTabs={leftTabs}
+                rightTabs={rightTabs}
+                activeItem={activeItem}
+            />
+
+            {(isLoadingVideos || isLoadingPodcasts) && <P>Loading...</P>}
```

Comment:
We might actually want to leave this out for now since it could look odd having articles loaded (since they are given at build time) then it still saying "loading...". I'll ask Dan about this but let's omit it for now if that is alright.